include/mbedtls/error_high.h

```diff
@@ -0,0 +1,375 @@
+/**
+ * \file error_high.h
```

Comment:
Why are the generated header files (`error_high.h`, `error_low.h`, `error_includes.h`, `version_features.h`) in `include/mbedtls` and not in `library`? When would a user of the library ever use them?

Reply:
As discussed IRL and for reference here, for a PoC I went for the easiest solution, re-using the existing favourable include order that only applies to `include/mbedtls`, but for a final version we could adapt the include path to allow for a private include dir with the same virtualisation property.